README.md

```diff
@@ -1,2 +1,159 @@
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://www.w3schools.com/html/">HTML</a></li>
+    <li><a href="https://www.w3schools.com/css/">CSS</a></li>
+  </ul>
+</details>
+
+<br />
+
+### Live Demo <a name="Live-server"></a>
+
+[WOMEN IN TECH WEBSITE](https://www.loom.com/share/85d6ee4122a74986895ca5dd81e36352)
+
+### Key Features <a name="key-features"></a>
+
+- *Fully responsive for all screen sizes*
+- *Social Media publicity*
+- Use the Figma design for the project😊👌
+- Use the semantics elements💯
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+- [ ] Clone the repo from here:
+  - https://github.com/JoyTaribagshaw/module-1-capstone-project 
+
+### Prerequisites
+
+In order to run this project you need:
+
+- You need a PC.
+- VS Code installed
+- A browser
+
+
+### Setup
+
+- Clone this repository from this link: 
+[Github](https://github.com/JoyTaribagshaw/module-1-capstone-project)
+- Navigate to the clone reppository by running cd `module-1-capstone-project`
+
+### Install
+
+Install this project with:
+
+  cd module-1-capstone-project-version
+  npm install
+
+
+### Usage
+
+To run the project, execute the following command:
+
+    open the index file in the browser
+
+  ### Run tests
+
+To run tests, click on inspect to open the browser dev tools where you can inspect the entire HTML and CSS code and the console for JavaScript code. 
+
+### Deployment
+
+You can deploy this project using:
+Github pages.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤 *Joy Tari*
+
+- GitHub: [Joy](https://github.com/JoyTaribagshaw)
+- Twitter: [Joy](https://twitter.com/JoyTariBagshaw)
+- LinkedIn: [Joy](https://www.linkedin.com/in/joy-tari-bagshaw-b8b891236?lipi=urn%3Ali%3Apage%3Ad_flagship3_profile_view_base_contact_details%3BlJIF9fsfQCO4UIewGNO%2B%2FQ%3D%3D)
+
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+**Add a registration page**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/JoyTaribagshaw/module-1-capstone-project/issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐ Show your support <a name="support"></a>
+
+If you like this project give it a ⭐
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to give a warm thank you to [Cindy Shin](https://www.behance.net/adagio07)
```

Comment:
- [optional] you can make it clear she is the author of the original design.

Reply:
Thank you so much for your review @tufoinnkuo10 , I have made it clear that she is the author.